pxi.py

```diff
@@ -39,6 +39,10 @@ class PXI:
 
     def __init__(self, address: Tuple[str, int]):
         self.logger = logging.getLogger(str(self.__class__))
+        self.logger.setLevel(logging.DEBUG)
```

Comment:
This level setting didn't work on the hybrid machine, no debug settings were output. Something we should take a look at